programs/fall-workshops.md

```diff
@@ -1,50 +1,155 @@
-
-This workshop will be on December 3rd from 9:45 am - 3 pm on the UC Berkeley campus. Please arrive at the Hearst Mining Circle on the UC Berkeley campus by 9:45! You must [register](/programs/fall-workshops-registration.html) to attend. **SIGNUPS CLOSE Wednesday, November 30th, 11:59 PM.** You CANNOT register after this time due to the fact that we have to order materials. Thank you for understanding.
+Are you interested in learning about gears as you test your skills in design?
+
+**PiE on the Road** is a traveling workshop developed by Pioneers in Engineering and hosted at local high schools to bring a new STEM experience to a high school near you.
+
+Our workshop focuses on a hands-on exploration of gears and how we utilize them with our custom gear kit. We’ll be exploring topics such as torque, gear ratios, and designing under constraints. We challenge high school students of all backgrounds to explore and innovate with us. Upon completing the workshop, students will be able to take their kit home to share and continue exploring.
+
+<!-- TODO: make visible later
+Each workshop has a total of 30 seats available. All students are welcome to attend any workshop, but we recommend the workshop at a high school closest to you. Students must register to attend. Should registration fill, we will place students on a waitlist in case a seat opens.
+-->
+
+PiE on the Road will be held at the following locations and times. Registration forms will be released shortly.
+
+Questions? Contact us at <mailto:education@pioneers.berkeley.edu>.
+
+<!--<h1 style="text-align: center">Events</h1>-->
+<!-- TODO: turn into cards -->
+
+<div class="pie-jumbotron blue-bkg">
+<div class="row">
+    <div class="col-xs-12"><h2>PiE on the Road @Hayward</h2></div>
+    <div class="col-sm-6">
+        <p>Location:<br>
+        Hayward High School<br>
+        1633 East Ave, Hayward, CA 94541<br>
+        Classroom to be announced
+        </p>
+        <p>Time:<br>
+        Saturday, September 30, 2017<br>
+        1 - 4 PM
+        </p>
+    </div>
+    <div class="col-sm-6">
+        <h2><a class="btn btn-primary darker-blue-bkg" style="font-size:16px" href="#" role="button">
+            Registration form coming soon
+            <!--Register for Hayward workshop &raquo;-->
+        </a></h2>
+        <!-- TODO: make visible later
+        <p>
+        PiE on the Road - Hayward Registration Form
+        Hayward Registration Deadline: Thursday, September 28, 2017 Midnight
+        </p>
+        -->
+    </div>
+</div>
+</div>
 
-Questions, comments, concerns? Email us at <mailto:education@pioneers.berkeley.edu>.
 
-<div class="text-center">
-    <a href="/programs/fall-workshops-registration.html" class="btn btn-primary">Register for Workshop</a>    
+<div class="pie-jumbotron">
+<div class="row">
+    <div class="col-xs-12"><h2>PiE on the Road @CCPA</h2></div>
```

Comment:
I'm not sure how many people will recognize CCPA for a heading - can you expand out to Coliseum College Prep Academy?

Reply:
addressed